ocs_ci/deployment/deployment.py

```diff
@@ -173,6 +173,15 @@ def deploy_cluster(self, log_cli_level="DEBUG"):
             setup_netsplit(
                 tmp_path, master_zones, worker_zones, x_addr_list, arbiter_zone
             )
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        if (
+            config.ENV_DATA.get("deploy_acm_hub_cluster")
+            and ocp_version >= version.VERSION_4_9
+        ):
+            try:
+                self.deploy_acm_hub()
```

Comment:
@prsurve what I meant here earlier was, you are in a multicluter environment, so for ex: there are 3 cluster contexts are present, before invoking deploy_acm_hub, you need to switc the context.

Reply:
yes you are right we need context switch here

Reply:
i will add in next comment

Reply:
i see ACM switch ctx is part of your pr hence we need your pr to be merged first

Reply:
done